### DISCUSSION TITLE
is this legit or thief?

### DISCUSSION THREAD

**Discussion by mygsunny** [onlaj/Piano-LED-Visualizer#445] [General]
Just want to know if @onlaj is affiliated with the selling of this software on https://pianoledshop.com/en-us
I'm making a video tutorial on this and just wanted to know if i should plug them or not.

**Comment by onlaj** [comment]
Hey.
Nope, they just used the code. I'm not affiliated with them in any way.

**Comment by grandgooroo** [comment]
Don't they have the obligation to quote the creator of the original program ?

**Reply by onlaj** [reply]
It's under the MIT license, so they can do whatever they want with it. However, if they were cool about it, I would mention them in the readme.

**Reply by mygsunny** [reply]
> It's under the MIT license, so they can do whatever they want with it. However, if they were cool about it, I would mention them in the readme.

i emailed them out of curiosity, and they said they came up with the software and have two patent on it. 😅

**Reply by onlaj** [reply]
Well, they are probably lying. The similarities are too great to be called a coincidence.

**Reply by grandgooroo** [reply]
Yep, the craziest thing is the promo they do in France saying that they are at the initiative of the project, they didn't even bother to customize the UI Design. If it wouldn't be a problem for you, I would have trolled their promos ^^.

**Reply by bbkiwi** [reply]
I thought the same. I looked at their patent document and it says "rejected" on the application. Anyway Ojlaj your work is fantastic!

**Comment by eage2005** [comment]
Seem like we can access the web interface with raspberrypi.local without knowing the IP address.

**Reply by onlaj** [reply]
In version 1.5 it will be "pianoledvisualizer.local" in case there are more Raspberries in the local network.

**Comment by sravel** [comment]
hello, 
I also found videos of the first version of the case which is exactly the 3D one of a pi zero!! 
I left a comment on their Facebook ad 3 times mentioning the fact that we found free code to do exactly the same thing but each time it was deleted!!!

**Comment by ztbishop** [comment]
Oddly enough I was about to order one of those Piano LED Plus before I found this project.  I had emailed their team in France with a list of questions on PC connectivity as their product claims to work with Synthesia.  
-Their only response was that they disabled compatibility with Synthesia / 3rd party apps to make a more seamless integration with their app....which I've read pushes on a subscription model, even for more than a few scale exercises.  
They did say to reach out with any other questions, after completely ignoring my additional questions about any limitations it has for importing your own midi files for those who don't wish to stay on subscription.  (something their site claims is possible).   So it leaves me wondering if they are crippling any of that functionality.  
I don't mind supporting a team that puts the work into building, but that was off-putting to me.   I saw one user comment on their youtube video that they are having trouble refunding after purchasing with the advertised expectation of working with Synthesia.
